app/src/main/java/io/rafaelribeiro/spendless/core/presentation/SpendLessDropDown.kt

```diff
@@ -80,22 +88,14 @@ fun <T>SpendLessDropDown(
     }
     Column(
         horizontalAlignment = Alignment.CenterHorizontally,
-        verticalArrangement = Arrangement.Center
+        verticalArrangement = Arrangement.Center,
+        modifier = modifier
     ) {
         Box(
             modifier = Modifier
                 .height(48.dp)
                 .fillMaxWidth()
-                .shadow(
-                    elevation = 32.dp,
-                    shape = RoundedCornerShape(16.dp),
-                    spotColor = DefaultShadowColor.copy(
-                        red = 24 / 255f,
-                        green = 0 / 255f,
-                        blue = 64 / 255f,
-                        alpha = 0.3f
-                    ),
-                )
+                .spendlessShadow()
```

Comment:
Created this helper modifier to add shadow to elements according to Figma design.